GliaWidgets/Sources/Layout/Layout+UIView.swift

```diff
@@ -0,0 +1,300 @@
+import UIKit
+
+extension UIView {
+
+    /// Defines available edges for establishing constratins.
+    ///
```

Comment:
I don't think this space is needed. Is there a reason all structs in this file have them?

Reply:
fixed  - 5966c5650a8fdf420523c179f24a8d4d0c741918